src/js/navigation.js

```diff
@@ -2,12 +2,54 @@ const navigationToggle = document.querySelector(".menu-btn");
 const navigation = document.querySelector("nav");
 const welcome = document.querySelector(".welcome");
 
-navigationToggle.addEventListener("click", ()  => {
+navigationToggle.addEventListener("click", () => {
   document.body.classList.toggle("nav-is-open");
+  welcome.classList.toggle("welcome-to-right");
   welcome.style.transition = "transform ease-out 250ms";
+  if (welcome.classList.contains("welcome-to-right")) {
+    const welcomeRight = document.querySelector(".welcome-to-right");
+    setTimeout(
+      () =>
```

Comment:
You are adding an eventListener every time you open the menu.
You can add it just once and have logic like.

"on-click"
if (nav is opened) {
  close nav
}

Reply:
misliš takole:

welcome.onclick = myFunction;

function myFunction() {
 if (body.classList.contains("nav-is-open") {
body.classList.remove("nav-is-open); itd.
}